recordset/recordset.js

```diff
@@ -78,10 +78,12 @@
         $scope.navbarBrand = (chaiseConfig['navbarBrand'] !== undefined? chaiseConfig.navbarBrand : "");
         $scope.navbarBrandImage = (chaiseConfig['navbarBrandImage'] !== undefined? chaiseConfig.navbarBrandImage : "");
         $scope.navbarBrandText = (chaiseConfig['navbarBrandText'] !== undefined? chaiseConfig.navbarBrandText : "Chaise");
-
+        var reloadCb = function() {
+                            window.location.reload();
+                        }; 
         $scope.login = function() {
```

Comment:
I don't think these login and logout function are even being used. You can completely remove these two functions.